recipe/meta.yaml

```diff
@@ -52,10 +60,11 @@ test:
 
     # Test for libraries.
     - test -f "${PREFIX}/lib/libfontconfig.a"
-    - test -f "${PREFIX}/lib/libfontconfig.dylib"  # [osx]
-    - test -f "${PREFIX}/lib/libfontconfig.so"     # [linux]
-    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
-    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+    - test -f "${PREFIX}/lib/libfontconfig.dylib"               # [osx]
+    - test -f "${PREFIX}/lib/libfontconfig.so"                  # [linux]
+
+    # Check macOS uses the system UUID.
+    - '! grep -q uuid "${PREFIX}/lib/pkgconfig/fontconfig.pc"'  # [osx]
```

Comment:
Not sure if we got this test right. Have tried replacing `uuid` with other things that are in `fontconfig.pc`, but it doesn't seem to fail. 🤔

Reply:
YAML picks up on `!` hence the quotes. `conda-build` may also be doing things.

Looking at [this line in the log]( https://travis-ci.org/conda-forge/fontconfig-feedstock/builds/403208154#L1587 ) seems that the `!` is dropped. 😞